ui/src/app/base/components/TooltipButton/TooltipButton.tsx

```diff
@@ -34,13 +38,14 @@ const TooltipButton = ({
     >
       <Button
         appearance="base"
-        className="u-display--block u-no-border u-no-margin"
+        aria-label={ariaLabel}
+        className="u-no-border u-no-line-height u-no-margin"
         hasIcon
         small
         type="button"
         {...buttonProps}
       >
-        {children || <Icon name="information" {...iconProps} />}
+        {children || <Icon name={iconName} {...iconProps} />}
```

Comment:
(This is out of scope of this PR but we should keep this in mind for later)

![Automatically sync images](https://user-images.githubusercontent.com/7452681/167107917-9c4c2e76-4c78-4e73-a8c1-b5884f679c66.png)


As the click/touch target is extremely small (often 16x16px) we might want to activate the tooltip on hover of not just the icon but the accompanying label as well (so in this case “Automatically sync images”). This raises some questions around the API of this component.

Should we always render the icon if the iconName prop is provided instead?

```suggestion
        {children}
        {iconName ? <Icon name={iconName} {...iconProps} /> : null}
```

If we wanted to keep this behaviour this should be made obvious (e.g. by conditional type that would return an error if you try to combine children and iconName).

Reply:
https://github.com/canonical-web-and-design/maas-ui/issues/3907

Reply:
+1 on being able to render both icon and children, and yeah thanks for filing an issue we can look into that later!